docs/content/getting-started/setup-openfga.mdx

```diff
@@ -397,53 +474,67 @@ or if you're using environment variables,
 docker run -p 8080:8080 -p 8081:8081 openfga/openfga -e OPENFGA_EXPERIMENTALS="feature1, feature2" run
 ```
 
-The following table table enumerates the experimental flags, a description of what they do, and the versions of OpenFGA the flag is supported in:
+The following table enumerates the experimental flags, a description of what they do, and the versions of OpenFGA the flag is supported in:
+
+| Name         | Description                                                        | OpenFGA Version       | Status                     |
+|--------------|--------------------------------------------------------------------|-----------------------|----------------------------|
+| otel-metrics | Enables support for exposing OpenFGA metrics through OpenTelemetry | `0.3.2 <= v < v0.3.5` | became default in `v0.3.5` |
+| list-objects | Enables ListObjects API                                            | `0.2.0 <= v < v0.3.3` | became default in `v0.3.3` |
 
-| name                   | description                                                                                              | openfga version |
-|------------------------|----------------------------------------------------------------------------------------------------------|-----------------|
-| otel-metrics           | Enables support for exposing OpenFGA metrics through OpenTelemetry | >= v0.3.2       |
 
 :::caution Warning
 Experimental features are not guaranteed to be stable and may lead to server instabilities. It is not recommended to enable experimental features for anything other than experimentation.
 
 Experimental feature flags are also not considered part of API compatibility and are subject to change, so please refer to each OpenFGA specific release for a list of the experimental feature flags that can be enabled for that release.
 :::
 
-## Telemetry (Metrics and Tracing)
+## Telemetry
+
+OpenFGA telemetry data is collected by default starting on version `v0.3.5`.
```

Comment:
I recommend a slightly modified exerbt at the top and then jump into each sub-section. I also recommend that we avoid going in to any OpenTelemetry Collector specific information (other than a reference/example in our docker-compose file). If a developer doesn't want to use the OpenTelemetry Collector that's a totally acceptable choice, and everything we've provided herein should be to the point and not OTEL specific.

Here's a slightly updated variation of this whole section. Let me know what you think?

# Telemetry
OpenFGA telemetry is exposed by default starting with release `v0.3.5`. The telemetry information that is exposed includes Metrics, Traces, and Logs.

:::note
Please refer to the [`docker-compose.yaml`](https://github.com/openfga/openfga/blob/main/docker-compose.yaml) file as an example of how to wireup Metrics and Tracing in OpenFGA in a Docker environment. This should serve as a good example that you can adjust for your specific deployment scenario.
:::

### Metrics
OpenFGA server metrics are exposed via Prometheus on `0.0.0.0:2112` and the `/metrics` endpoint (by default). They are enabled by default, but can be disabled with the `--metrics-enabled=false` flag (`OPENFGA_METRICS_ENABLED=false` env variable).

To configure an alternative metrics server address, please provide the `--metrics-addr` flag (`OPENFGA_METRICS_ADDR` env variable). For example,

```
openfga run --metrics-addr=0.0.0.0:2114
```

You may also provide the `--metrics-enable-rpc-histograms` flag (`OPENFGA_METRICS_ENABLE_RPC_HISTOGRAMS` env variable) to enable per-endpoint distribution metrics. This is helpful if you want to see the request latency per-endpoint of your OpenFGA deployment.

## Tracing
OpenFGA supports tracing through the [OpenTelemetry Tracing API](https://opentelemetry.io/docs/reference/specification/trace/api/). Tracing is disabled by default, but you can enable it with the `--trace-enabled=true` flag (`OPENFGA_TRACE_ENABLED=true` env variable).

By default traces are exported to the configured `--trace-otlp-endpoint` (`OPENFGA_TRACE_OTLP_ENDPOINT` env variable) in the OpenTelemetry [`otlp`](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/otlp.md) format. To increase or decrease the trace sampling ratio, you can provide the `--trace-sample-ratio` flag (`OPENFGA_TRACE_SAMPLE_RATIO` env variable).

:::caution Warning
It is not recommended to sample all traces (e.g. `--trace-sample-ratio=1`). In practice, you'll need to set your sampling ratio based on the amount of traffic your deployment receives. Higher traffic will require less sampling and lower traffic can tolerate higher sampling ratios.
:::

## Logging
OpenFGA produces structured logs to stdout, and it can be configured with the following flags:
* `--log-format` - sets the log format. Today we support `text` and `json` format.
* `--log-level` - sets the minimum log level (defaults to 'info'). For more information on the supported levels please refer to the help output of the OpenFGA server (e.g. `openfga run -h`).

Reply:
Thanks for this! I applied your suggestions.